### ISSUE TITLE
Homepage - Image missing alt attribute

### ISSUE THREAD

**Issue by DavidKohav** [DrupalHub/drupalhub#238]
Web Content Accessibility Guidelines (WCAG) 2.0, Level A: 1.1.1 Non-text Content

All images must have an alt attribute. Not supplying an alt attribute will mean that users who cannot see the image will not understand what the image conveys.

```<img src="img/logo.png">```
